examples/lazy-input/main.rs

```diff
@@ -77,11 +81,12 @@ trait Db: salsa::Database {
 }
 
 #[salsa::db]
+#[derive(Clone)]
```

Comment:
Nit: I kind of preferred the old salsa's `ParallelDatabase` trait rather than relying on `Clone`. It makes the contract more explicit.

Reply:
At least in rust-analyzer's case, I can't really think of a situation where it _wouldn't_ make sense to have every `Database` be a `ParallelDatabase`. I don't think we'd want to make everything parallel, but having the option to do so without extensive refactors is really valuable, in my view.